### DISCUSSION TITLE
posts/my-blog-design-journey

### DISCUSSION THREAD

**Discussion by giscus** [poodlepoodle/poodle-blog-comments-giscus#1] [Comments]
# posts/my-blog-design-journey

새로운 기술이 파도처럼 몰려와도 지워지지 않을 개발자국을 남깁니다.

https://poodlepoodle.me/blog/my-blog-design-journey

**Comment by ShipFriend0516** [comment]
다음 글 빨리 써주세요 제발

**Comment by chajju** [comment]
블로그 디자인 레퍼런스를 찾다가 너무나도 유익한 글을 보고 갑니다..❤️ 
혹시 와이어프레임이랑 디자인 시스템 설계하는데 레퍼런스로 활용해도 될까요 ㅎㅎ !?

**Reply by poodlepoodle** [reply]
안녕하세요~! 편하게 활용해주셔도 됩니다!! 😀
